spec/controllers/redirections_controller_spec.rb

```diff
@@ -16,7 +16,7 @@
     describe "GET #{action}" do
       it "creates a Redirection if it doesn't exist" do
         new_slug = "new"
-        url = "http://example.com"
+        url = "http://notareal.domain"
```

Comment:
Thanks for fixing the tests!

Reply:
Oh wait, actually – could you revert these spec changes since they no longer are needed (I think)? I believe these changes should be folded into your future PR disallowing `example.com`.